release-team/hack/sync-bug-triage-github-project-beta.sh

```diff
@@ -0,0 +1,228 @@
+                    query($org: String!, $number: Int!) {
+                    organization(login: $org){
+                        projectV2(number: $number) {
+                        id
+                        fields(first:100) {
+                            nodes {
+                            ... on ProjectV2Field {
+                                id
+                                name
+                            }
+                            ... on ProjectV2SingleSelectField {
+                                id
+                                name
+                                options {
+                                    id
+                                    name
+                                }
+                            }
+                            }
+                        }
+                        }
+                    }
+                    }' -f org=${ORGANIZATION} -F number=${PROJECT_NUMBER} --jq "$query")"
+    echo $ID
+}
+
+
+function add_items_to_github_beta_project {
+    	    gh api graphql -f query='
+                    mutation (
+                        $project: ID!
+                        $item: ID!
+                        $type_field: ID!
+                        $option_id: String!
+                    ) {
+                        set_issue_type: updateProjectV2ItemFieldValue(input: {
+                            projectId: $project
+                            itemId: $item
+                            fieldId: $type_field
+                            value: {
+                                singleSelectOptionId: $option_id
+                            }
+                        }) {
+                            projectV2Item {
+                                id
+                            }
+                        }
+                    }' -f project="${PROJECT_ID}" -f item="${1}"  -f type_field="${TYPE_FIELD_ID}" -f option_id="${2}" --silent
+}
+
+echo 'Starting sync...'
+echo -e "[INFO] Fetching the list of open issues and PRs from k/k under the current release milestone, ${MILESTONE}"
+
+## Fetch issues
+MILESTONE_ISSUES_JSON="$(gh api graphql --paginate -f query='
+            query($org: String!, $repo: String!, $milestone: String!){
+              repository(owner: $org, name: $repo) {
+                description
+                url
+                milestones(states: [OPEN],first:1, query: $milestone) {
+                  nodes{
+                    issues(states:[OPEN], first: 200){
+                      nodes{
+                        id
+                      }
+                      pageInfo{
+                        hasNextPage
+                        endCursor
+                      }
+                    }
+                  }
+                  pageInfo{
+                    hasNextPage
+                    endCursor
+                  }
+                }
+              }
+            }' -f org=${ORGANIZATION} -f repo=${REPOSITORY} -f milestone="${MILESTONE}")"
```

Comment:
```suggestion
            }' -f org="${ORGANIZATION}" -f repo="${REPOSITORY}" -f milestone="${MILESTONE}")"
```
Adding double quotes to the other vars.

Reply:
addressed. Thank you!